Assets/Scripts/UI/Leaderboard/UILeaderboardMenu.cs

```diff
@@ -335,5 +378,47 @@ private void SetCurrentLeaderboardDescription()
             CurrentSelectedLeaderboardTxt.text = $"{currentSelectedDefinitionLeaderboardId} - " +
                 $"{currentSelectedDefinitionStatName} - {Enum.GetName(typeof(LeaderboardGroup), currentGroup)}";
         }
+
+        /// <summary>
+        /// If a leaderboard is selected, re-query and re-display its values.
+        /// </summary>
+        public void RefreshCurrentSelectedLeaderboard()
```

Comment:
I tend to think that a clearer way to organize this would be to have a `static` method that accepts a leader board as a parameter. This works perfectly fine, but the flexibility of the `static` implementation expands the ways it can be leveraged. Not sure whether that is a correct assessment or whether it is more subjective. Not something I think needs changing perse - just musing.